lab-server/CONTRIBUTING.md

```diff
@@ -44,10 +44,10 @@ by using the following command: `docker compose --profile local up`
         "access-type" should be "bearer only". In the credentials tab you need
         to create a secret and update the value `KEYCLOAK_SECRET` accordingly
       - For the frontend the name should be "pharme-app". The "access-type"
-        should be set to "public"
+        should be set to "public" and the redirect URI should be `*`.
```

Comment:
Technically '*' redirect URI is bad practice security-wise. Maybe add a sentence underlining this fact + that this should only be done in a local testing environment

Reply:
Added in [a011e07](https://github.com/hpi-dhc/PharMe/pull/316/commits/a011e074815948b3dd1d09a71c1c2952b8bc8c87).